chainregistry/config.go

```diff
@@ -0,0 +1,77 @@
+package chainregistry
+
+import (
+	"time"
+
+	"github.com/btcsuite/btcwallet/wallet"
+	"github.com/lightninglabs/neutrino"
+	"github.com/lightningnetwork/lnd/channeldb"
+	"github.com/lightningnetwork/lnd/lncfg"
+)
+
+// Config houses necessary fields that a ChainControl instance need to
+// function.
+type Config struct {
```

Comment:
just move this config above the `ChainControl` definition?

Reply:
I did this in some other packages, should I change those instances too?

Reply:
Yeah, I don't think we usually makes the config its own file.